bindings/web/README.md

```diff
@@ -1,67 +1,53 @@
-<div align="center">
+# libparsec-electron-bindings
 
-  <h1><code>wasm-pack-template</code></h1>
+Remember to have [`wasm-pack`](../../docs/development/README.md#base-requirement) installed !
 
-  <strong>A template for kick starting a Rust and WebAssembly project using <a href="https://github.com/rustwasm/wasm-pack">wasm-pack</a>.</strong>
-
-  <p>
-    <a href="https://travis-ci.org/rustwasm/wasm-pack-template"><img src="https://img.shields.io/travis/rustwasm/wasm-pack-template.svg?style=flat-square" alt="Build Status" /></a>
-  </p>
-
-  <h3>
-    <a href="https://rustwasm.github.io/docs/wasm-pack/tutorials/npm-browser-packages/index.html">Tutorial</a>
-    <span> | </span>
-    <a href="https://discordapp.com/channels/442252698964721669/443151097398296587">Chat</a>
-  </h3>
-
-  <sub>Built with 🦀🕸 by <a href="https://rustwasm.github.io/">The Rust and WebAssembly Working Group</a></sub>
-</div>
+```shell
+curl https://rustwasm.github.io/wasm-pack/installer/init.sh -sSf | sh
+```
 
-## About
+## Build
 
-[**📚 Read this template tutorial! 📚**][template-docs]
+tl;dr: `../../make.py wr`
 
-This template is designed for compiling Rust libraries into WebAssembly and
-publishing the resulting package to NPM.
+```shell
+# First run (to install dependencies)
+../../make.py web-dev-install
+# or just
+../../make.py wi
+# Subsequent runs
+../../make.py web-dev-rebuild
+# Or just
+../../make.py wr
+```
```

Comment:
Same as above
````suggestion
First run (to install dependencies)
```shell
../../make.py wi    # wi is an alias for web-dev-install
```

Subsequent runs
```shell
../../make.py wr    # wr is an alias for web-dev-rebuild
```
````